backend/src/server/routes/v3/certificates-router.ts

```diff
@@ -343,4 +343,142 @@ export const registerCertificatesRouter = async (server: FastifyZodProvider) =>
       return data;
     }
   });
+
+  server.route({
+    method: "POST",
+    url: "/:certificateId/renew",
+    config: {
+      rateLimit: writeLimit
+    },
+    schema: {
+      hide: false,
+      tags: [ApiDocsTags.PkiCertificates],
+      params: z.object({
+        certificateId: z.string().uuid()
+      }),
+      response: {
+        200: z.object({
+          certificate: z.string().trim(),
+          issuingCaCertificate: z.string().trim(),
+          certificateChain: z.string().trim(),
+          privateKey: z.string().trim().optional(),
+          serialNumber: z.string().trim(),
+          certificateId: z.string()
+        })
+      }
+    },
```

Comment:
IGNORE PLACEMENT OF COMMENT

I think this might be better termed as "Auto-renew days before expiry"... I find it more intuitive? 

<img width="929" height="444" alt="image" src="https://github.com/user-attachments/assets/5f72241e-e113-43b7-a14f-619e755b22ea" />